data/json/npcs/classes.json

```diff
@@ -97,7 +97,7 @@
     "type": "npc_class",
     "id": "NC_COWBOY",
     "name": { "str": "Cowboy" },
-    "job_description": "Just looking for some wrongs to right.",
+    "job_description": "I'm looking to make an honest living using the skills I gained in the country.",
```

Comment:
This line doesn't actually show up anywhere afaik

Reply:
Not for players, but on the off chance someone wants to use some NPCs somewhere, they might haphazardly spawn a couple of these thinking they're lawful good or something. It doesn't really matter in the end, so eh.